srcpkgs/igt-gpu-tools/patches/i-g-t-Fix-global-symbol-loading-failure-in-resolve-function.patch

```diff
@@ -0,0 +1,63 @@
+diff --git a/lib/igt_halffloat.c b/lib/igt_halffloat.c
```

Comment:
Do you have an upstream commit for this?

Reply:
Not yet, the upstream patch seems to haven't been merged into master. BTW, I attach the link about the patch.